backend/src/apiserver/resource/resource_manager.go

```diff
@@ -368,7 +369,7 @@ func (r *ResourceManager) CreatePipelineAndPipelineVersion(p *model.Pipeline, pv
 	if pipelineSpecURI != "" {
 		pv.PipelineSpecURI = pipelineSpecURI
 	}
-	tmpl, err := template.New(pipelineSpecBytes)
+	tmpl, err := template.New(pipelineSpecBytes, !r.options.CacheDisabled)
```

Comment:
I may have misunderstood but I think this is the correct way
```suggestion
	tmpl, err := template.New(pipelineSpecBytes, r.options.CacheDisabled)
```